src/libs/Notification/PushNotification/subscribePushNotification/index.ts

```diff
@@ -1,8 +1,113 @@
 import Onyx from 'react-native-onyx';
-import subscribeToReportCommentPushNotifications from '@libs/Notification/PushNotification/subscribeToReportCommentPushNotifications';
+import applyOnyxUpdatesReliably from '@libs/actions/applyOnyxUpdatesReliably';
+import * as ActiveClientManager from '@libs/ActiveClientManager';
+import Log from '@libs/Log';
+import Navigation from '@libs/Navigation/Navigation';
+import type {ReportActionPushNotificationData} from '@libs/Notification/PushNotification/NotificationType';
+import getPolicyEmployeeAccountIDs from '@libs/PolicyEmployeeListUtils';
+import {extractPolicyIDFromPath} from '@libs/PolicyUtils';
+import {doesReportBelongToWorkspace, getReport} from '@libs/ReportUtils';
+import Visibility from '@libs/Visibility';
+import * as Modal from '@userActions/Modal';
+import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+import type {OnyxUpdatesFromServer} from '@src/types/onyx';
+import {isEmptyObject} from '@src/types/utils/EmptyObject';
 import PushNotification from '..';
 
+let lastVisitedPath: string | undefined;
+Onyx.connect({
+    key: ONYXKEYS.LAST_VISITED_PATH,
+    callback: (value) => {
+        if (!value) {
+            return;
+        }
+        lastVisitedPath = value;
+    },
+});
+
+function getLastUpdateIDAppliedToClient(): Promise<number> {
+    return new Promise((resolve) => {
+        Onyx.connect({
+            key: ONYXKEYS.ONYX_UPDATES_LAST_UPDATE_ID_APPLIED_TO_CLIENT,
+            callback: (value) => resolve(value ?? 0),
+        });
+    });
+}
+
+function applyOnyxData({reportID, reportActionID, onyxData, lastUpdateID, previousUpdateID}: ReportActionPushNotificationData): Promise<void> {
```

Comment:
Additional function comments would be useful here

Reply:
I thought about it, but I'm not sure there's anything to comment on. "Apply the onyx data" seems like a useless comment and we didn't have any comments for it before either